components/scream/src/physics/p3/p3_constants.hpp

```diff
@@ -0,0 +1,63 @@
+#ifndef P3_CONSTANTS_HPP
+#define P3_CONSTANTS_HPP
+
+#include "share/scream_types.hpp"
+
+#include <vector>
+
+namespace scream {
+namespace p3 {
+
+/*
+ * Mathematical constants used by p3.
+ */
+
+template <typename Scalar>
+struct Constants
+{
+  static constexpr Scalar INV_RHOW = 1.e-3;
+  static constexpr Scalar RHOW     = 997.0;
+  static constexpr Scalar THRD     = 1.0/3.0;
```

Comment:
Most of these constants came from the fortran p3 implementation. I agree, some of the names are a bit weak.

> Besides, is there a reason why we have a name for 1/3, rather than writing `blah/3.0` in the code?

I think this is another fortranism.